### ISSUE TITLE
Insert update into a different table

### ISSUE THREAD

**Issue by jenunifer** [jasonhinkle/phreeze#88]
Hi Jason!

I am trying update a database field with user input on a ListView.tpl page of another table. 
Here is what i have:
Table1  Table2

 I did a join in the Table1Reporter.php and can view the field I need from Table2 in the Table1ListView.tpl pages html table and model.  I need to be able to get the users input from Table2's field in view on Table1ListView.tpl model,  back into the Table2 field. 
Basically, I need to update Table2 field from Table1's list view page.  Best method?

Thanks!
